### PR TITLE
Feature css expectations

### PR DESCRIPTION
**Pull request by aguspina**
It's been a while since my last commit, please be nice 🤕 with me

You can now do the following things

```
css:p DeclaresStyle: color:blue
css:h2 DeclaresStyle: color
css:font DeclaresStyle
```

### CODE REVIEW

**Comment by aguspina** [comment]
@flbulgarelli @julian-berbel Please review again

File: spec/expectations_spec.rb [hunk 1]
```diff
@@ -68,4 +68,29 @@ def compile_and_run(request)
...
+
+  describe 'body DeclaresStyle' do
+    let(:code) { "p, h2 {color: blue; font-size: 4px;} div.cuadrado circulo {background: red}" }
+    let(:expectations) { [
+        {binding: 'css:h2', inspection: 'DeclaresStyle'},
+        {binding: 'css:div.cuadrado circulo', inspection: 'DeclaresStyle'} ] }
+
+    it { expect(result).to eq [
+        { expectation: {binding: 'css:h2', inspection: 'DeclaresStyle'}, result: true},
+        { expectation: {binding: 'css:div.cuadrado circulo', inspection: 'DeclaresStyle'}, result: true} ] }
```
Comment (flbulgarelli) [inline spec/expectations_spec.rb]: It would be nice to also test the negative scenarios where expectation fails
